src/ripple/protocol/impl/Seed.cpp

```diff
@@ -57,10 +60,38 @@ Seed::Seed (uint128 const& seed)
 
 //------------------------------------------------------------------------------
 
+// Stack buffer whose memory is securely zeroed in the destructor
+template <class T, size_t N>
+struct zero_after_use_buf : std::array<T, N>
+{
+    using std::array<T, N>::array;
+    zero_after_use_buf() = default;
+    zero_after_use_buf(zero_after_use_buf const& rhs) = default;
+    ~zero_after_use_buf()
+    {
+        beast::secure_erase(this->data(), sizeof(T) * this->size());
+    }
+};
+struct zero_after_use_uint256 : uint256
+{
+    using uint256::uint256;
+    zero_after_use_uint256() = default;
+    zero_after_use_uint256(zero_after_use_uint256 const& other) : uint256{other}
```

Comment:
= default?

Reply:
Addressed in 91c6c712b0973af30980b849d50e33a4f715d449